RestSharp/Authenticators/OAuth/OAuthTools.cs

```diff
@@ -34,93 +24,85 @@ internal static class OAuthTools
 
         private static readonly object randomLock = new object();
 
-#if !SILVERLIGHT && !WINDOWS_PHONE && !WINDOWS_UWP
         private static readonly RandomNumberGenerator rng = RandomNumberGenerator.Create();
-#endif
+
+        /// <summary>
+        ///     All text parameters are UTF-8 encoded (per section 5.1).
+        /// </summary>
+        /// <seealso cref="http://www.hueniverse.com/hueniverse/2008/10/beginners-gui-1.html" />
```

Comment:
This link is dead.

Reply:
Sorry, not my fault. It was there and I have done nothing.